packages/e2e-tests/plugins/interactive-blocks/directive-context/view.js

```diff
@@ -88,3 +88,21 @@ const { actions } = store( 'directive-context-navigate', {
 		},
 	},
 } );
+
+store( 'directive-context-watch', {
+	actions: {
+		increment: () => {
+			const ctx = getContext();
+			ctx.counter = ctx.counter + 1;
+		},
+	},
+	callbacks: {
+		countChanges: () => {
+			const ctx = getContext();
+			// Subscribe to changes in counter.
+			// eslint-disable-next-line no-unused-expressions
+			ctx.counter;
```

Comment:
Why is counter called here?

Reply:
It's mentioned in the comment above. The idea is to subscribe to changes in `ctx.counter`, even though the value is not used. The callback runs when `ctx.counter` is updated.